niimpy/EDA/EDA_countplot.py

```diff
@@ -149,38 +213,47 @@ def EDA_countplot(df, fig_title, plot_type = 'count', points = 'outliers', aggre
     assert ((isinstance(user,str)) or (user is None)), "user is not a string or None type."
     assert ((isinstance(column,str)) or (column is None)), "column in not a string or None type."
     
+    # Plot counts
     if plot_type == 'count':
         if aggregation == 'group':
-            n_events = df[['group', 'user']].groupby(['user', 'group']).size().to_frame()
-            n_events.columns = ['values']
-            n_events = n_events.reset_index()
-            EDA_boxplot_(n_events,
-                         fig_title,
-                         points, 
-                         y = 'values',
-                         xlabel="Group",
-                         ylabel="Count")
+                       
+            n_events = get_counts(df,aggregation)
+            print(n_events)
+                   
+            fig = EDA_boxplot_(n_events,
+                               fig_title,
+                               points,
+                               y = 'values',
+                               xlabel="Group",
+                               ylabel="Count",
+                               binning=binning
+                               )
         
         elif aggregation == 'user':
-            n_events = df[['user']].groupby(['user']).size().to_frame()
-            n_events.columns = ['values']
-            n_events = n_events.reset_index()
-            EDA_barplot_(n_events, 
-                        fig_title, 
-                        xlabel="User",
-                        ylabel="Count")
+                       
+            n_events = get_counts(df,aggregation)
+            
+            fig = EDA_barplot_(n_events, 
+                               fig_title, 
+                               xlabel="User",
+                               ylabel="Count")
         
         else:
             pass
         
+    # Plot values
     elif plot_type == 'value':
         if aggregation == 'group':
-            EDA_boxplot_(df,
-                        fig_title,
-                        points,
-                        y=column,
-                        xlabel="Group",
-                        ylabel="Value")
+            fig = EDA_boxplot_(df,
+                               fig_title,
+                               points,
+                               y=column,
+                               xlabel="Group",
+                               ylabel="Value",
+                               binning=binning
+                               )
         
     else:
-        pass
\ No newline at end of file
+        pass
+    
+    return fig
```

Comment:
```suggestion
    return fig
    
```
(Add new line at the end)